db/migrate/20190725082404_add_ldu_and_team_to_case_information.rb

```diff
@@ -0,0 +1,8 @@
+class AddLduAndTeamToCaseInformation < ActiveRecord::Migration[5.2]
+  def change
+    change_table :case_information do |t|
+      t.string :ldu
```

Comment:
Are these (ldu and team) just the IDs?

Reply:
These are the names - not sure that the IDs are useful..? Actually I don't know which ones I am meant to be bringing in.

Reply:
The problem with the names is that they're inconsistent in the data file, for instance Gwent appears as "Gwent" and "Gwent LDU".  Once we get to https://dsdmoj.atlassian.net/browse/POM-187 we'll need to be able to get the email address of the LDU.  Maybe we should worry about that once we know what the email addresses are keyed against?

Reply:
It's not clear what we want to do here. We're not doing anything with the data once we bring it in - we're just preventing allocations unless it has something?

Reply:
I think one of the pages might have it used in display (prisoner profile?) but that's probably a later thing.